docs/_sidebar.md

```diff
@@ -65,6 +65,7 @@
         - [Redis](en/component_specs/sequencer/redis.md)
         - [Zookeeper](en/component_specs/sequencer/zookeeper.md)
         - [MongoDB](en/component_specs/sequencer/mongo.md)
+      - [Secret Store](en/component_specs/secret/common.md)
```

Comment:
Please make sure it's aligned with `Sequencer`
![image](https://user-images.githubusercontent.com/26001097/166855366-2a1b6633-2e7a-436b-8ed3-b411e9d00fc5.png)